src/viur/core/prototypes/list.py

```diff
@@ -258,7 +258,7 @@ def edit(self, key: db.Key | int | str, *args, **kwargs) -> t.Any:
         if (
             not kwargs  # no data supplied
             or not current.request.get().isPostRequest  # failure if not using POST-method
-            or not skel.fromClient(kwargs)  # failure on reading into the bones
+            or not skel.fromClient(kwargs, amend=True)  # failure on reading into the bones
```

Comment:
Hello @sveneberth,

the default amending in the edit-action does not deactivate or bypass any validation. Can you give me a concrete example where this might happen? The advantage of this implementation is, that you can either pass all bones or only a subset which is changed.

Reply:
If _amend_ is enabled, you can omit fields that should be set, because they're `required`: 
https://github.com/viur-framework/viur-core/blob/9d7811832956d51a0ae6e70215c6fa2f543d2b5c/src/viur/core/skeleton.py#L496C41-L496C107

This effectively makes `ReadFromClientErrorSeverity.NotSet` superfluous.

Reply:
Okay, then leave it like that and we'll just keep an eye on it.